spp_event_demo/models/house_visit.py

```diff
@@ -0,0 +1,20 @@
+# Part of OpenSPP. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models
+
+
+class OpenSPPHouseVisit(models.Model):
+    _name = "spp.event.house.visit"
+    _description = "House Visit"
+
+    summary = fields.Char()
+    description = fields.Text()
```

Comment:
remove `description` and add the fields:
- is_farm
- farm_size_acre
- photo
- number_of_pigs
- number_of_cows
- no_food_stock
- disabled